spark/v3.5/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestDelete.java

```diff
@@ -1301,7 +1305,7 @@ public void testDeleteWithMultipleSpecs() {
     } else if (mode(table) == MERGE_ON_READ && formatVersion >= 3) {
       validateMergeOnRead(currentSnapshot, "3", "4", null);
     } else {
-      validateMergeOnRead(currentSnapshot, "3", "3", null);
```

Comment:
nit, previous branch is no longer needed

Reply:
ah yes, fixed!